cursorless-talon/src/get_grapheme_spoken_form_entries.py

```diff
@@ -32,7 +34,7 @@ def generate_lists_from_capture(capture_name) -> Iterator[str]:
     if capture_name.startswith("self."):
         capture_name = "user." + capture_name[5:]
     try:
-        rule = registry.captures[capture_name][0].rule.rule
+        rule = registry.captures[capture_name][-1].rule.rule
```

Comment:
-1/last is actually the active one. This is a bug on man that could potentially use the wrong spoken forms